homeassistant/components/plugwise/coordinator.py

```diff
@@ -47,12 +52,16 @@ async def _async_update_data(self) -> PlugwiseData:
         """Fetch data from Plugwise."""
         try:
             data = await self.api.async_update()
-        except XMLDataMissingError as err:
+        except (InvalidXMLError, ResponseError) as err:
             raise UpdateFailed(
-                f"No XML data received for: {self.api.smile_name}"
+                "Invalid XML data, or error indication received for the Plugwise Adam/Smile/Stretch"
             ) from err
-        except PlugwiseException as err:
-            raise UpdateFailed(f"Updated failed for: {self.api.smile_name}") from err
+        except UnsupportedDeviceError as err:
+            raise UpdateFailed(
+                "Unsupported device found, please create an Issue in the Home Assistant Core github"
```

Comment:
Adding support for new devices is considered a feature request. For these things, people should not raise issues (they will be closed).

Reply:
This error is raised in the case that we codeowners miss a major firmware update for the existing supported devices, with potentially breaking changes.
Since Tom and I are on the beta-list of Plugwise this should not happen often :)

Reply:
Sure, but the review comment remains the same.